crates/re_data_ui/src/data.rs

```diff
@@ -173,40 +173,6 @@ impl DataUi for re_types::datatypes::Vec3D {
     }
 }
 
-impl DataUi for Rect2D {
-    fn data_ui(
-        &self,
-        _ctx: &mut ViewerContext<'_>,
-        ui: &mut egui::Ui,
-        _verbosity: UiVerbosity,
-        _query: &re_arrow_store::LatestAtQuery,
-    ) {
-        ui.label(match self {
-            Rect2D::XYWH(LegacyVec4D([top, left, width, height]))
-            | Rect2D::YXHW(LegacyVec4D([left, top, height, width])) => {
-                format!("top: {top}, left: {left}, width: {width}, height: {height}")
-            }
-            Rect2D::XYXY(LegacyVec4D([left, top, right, bottom]))
-            | Rect2D::YXYX(LegacyVec4D([top, left, bottom, right])) => {
-                format!("top: {top}, left: {left}, right: {right}, bottom: {bottom}")
-            }
-            Rect2D::XCYCWH(LegacyVec4D([center_x, center_y, width, height])) => {
-                format!(
-                    "center: {}, width: {width}, height: {height}",
-                    LegacyVec2D([*center_x, *center_y])
-                )
-            }
-            Rect2D::XCYCW2H2(LegacyVec4D([center_x, center_y, half_width, half_height])) => {
-                format!(
-                    "center: {}, half-width: {half_width}, half-height: {half_height}",
-                    LegacyVec2D([*center_x, *center_y])
-                )
-            }
-        })
-        .on_hover_text(format!("area: {}", self.width() * self.height()));
```

Comment:
This highlights that we want to have ui:s tailored for _archetypes_ pretty soon.

Reading a box that was logged as `XYWH` as center and half-width is not a great user experience. On slack we planned on presenting a ui where the user can chose how to view their logged boxes

Reply:
* I added a note to https://github.com/rerun-io/rerun/issues/3231